account_banking_reconciliation/views/account_banking_reconciliation.xml

```diff
@@ -54,7 +54,7 @@
                                placeholder="Enter Ending Balance"/>
                         <field name="last_ending_date"
                                placeholder="Last Statement Date"
-                               readonly="1"/>
+                               readonly="1" force_save="1"/>
```

Comment:
Was it working earlier ?
As we added `force_save` we have to make sure we are not doing this in create/write also